docs/source/usage.rst

```diff
@@ -0,0 +1,364 @@
+====================
+Basic Usage Examples
+====================
+
+For the examples on this page, we assume you've :doc:`installed the bigchaindb_driver JavaScript package <quickstart>`,
+and you have determined the BigchainDB Root URL (issue: move this to general docs)
+of the node or cluster you want to connect to.
+
+This example guides you through creating and transferring an asset.
+We walk through the code explaining its use, some pieces are left out
+because they have no real use (e.g. defenition of global variable)
+*Full working code* can be found at the bottom of this document.
+The following code are just snippets.
+
+Getting Started
+---------------
+We begin by creating an object of BigchainDB driver:
+
+.. code-block:: js
+
+	const driver = require('bigchaindb-driver')
+
+Next, we define a constant containing the API path.
+
+.. code-block:: js
+
+	const API_PATH = 'http://localhost:9984/api/v1/'
+
+Create Connection With BigchainDB
+---------------------------------
+
+A simpel connection with BigchainDB can be established like this.
+
+.. code-block:: js
+
+	const conn = new driver.Connection(API_PATH)
+
+It is also possible to connect to a BigchainDB node of the IPDB test net.
+To do so, you need to pass the **app_id and app_key**. 
+
+.. code-block:: js 
+
+	let bdb = new driver.Connection('https://test.ipdb.io/api/v1/', { 
+		app_id: 'dgi829l9',
+		app_key: 'u008ik1bf83b43ce3a95uu0727e66fb9'
+	})
+
+Cryptographic Identities Generation
+-----------------------------------
+Alice and Bob are represented by public/private key pairs. The private key is
+used to sign transactions, meanwhile the public key is used to verify that a
+signed transaction was indeed signed by the one who claims to be the signee.
+
+.. code-block:: js
+
+	const alice = new driver.Ed25519Keypair()
+	const bob = new driver.Ed25519Keypair()
+
+Digital Asset Definition
+------------------------
+
+As an example, let’s consider the creation and transfer of a digital asset
+that represents a bicycle:
+
+.. code-block:: js
+
+	assetdata = {
+		'bicycle': {
+			'serial_number': 'abcd1234',
+			'manufacturer': 'Bicycle Inc.',
+		}
+	}
+
+We'll suppose that the bike belongs to Alice, and that it eventually will be 
+transferred to Bob.
+
+In general, you are free to define any JSON object you which to store for the 
+``'data'`` property (assetdata).
+
+Metadata Definition (*optional*)
+--------------------------------
+
+You can `optionally` add metadata to a transaction. Any JSON object is accepted.
+
+For example, the bicycle will be transferred on earth which is metadata:
+
+.. code-block:: js
+
+	metadata = {'planet': 'earth'}
+
+Asset Creation
+--------------
+
+We're now ready to create the digital asset. First, let's make a 'CREATE' 
+transaction:
+
+.. code-block:: js
+	
+	const txCreateAliceSimple = driver.Transaction.makeCreateTransaction(
+   		assetdata,
+   		metadata,
+
+		// A transaction needs an output
+		[ driver.Transaction.makeOutput(
```

Comment:
I think you should explain between l104 - l107 what inputs and outputs in a BigchainDB transaction are.